x/blob/types/payforblob.go

```diff
@@ -140,6 +140,22 @@ func CreateCommitment(namespace []byte, blobData []byte, shareVersion uint8) ([]
 	return merkle.HashFromByteSlices(subTreeRoots), nil
 }
 
+// CreateMultiShareCommitment generates a commitment over multiple blobs at
+// arbitrary points in the square. It uses the normal commitment creation
+// function per blob, and then creates a merkle root of those commitments.
+func CreateMultiShareCommitment(nsids [][]byte, blobs [][]byte, shareVersions []uint32) ([]byte, error) {
```

Comment:
Why uint32 here for shareVersions?

Reply:
protobuf smallest supported uint is uint32, so that's what we end up using in many places